tools/tileGet.py

```diff
@@ -82,17 +88,29 @@ def retrieveMapServerTiles(options, west, south, east, north, decals, net):
         sx, sy = fromLatLonToTile(north, west, zoom)
         ex, ey = fromLatLonToTile(south, east, zoom)
         numTiles = (ex - sx + 1) * (ey - sy + 1)
-    for x in range(sx, ex + 1):
-        for y in range(sy, ey + 1):
-            request = "%s/%s/%s/%s" % (options.url, zoom, y, x)
-#            print(request)
-#            opener = urllib.build_opener()
-#            opener.addheaders = [('User-agent', 'Mozilla/5.0')]
-#            urllib.install_opener(opener)
-            filename = os.path.join(options.output_dir, "%s%s_%s.jpeg" % (options.prefix, x, y))
-            urllib.urlretrieve(request, filename)
-            if os.stat(filename).st_size < options.min_file_size:
-                raise ValueError("small file")
+        
+    # opener = urllib.build_opener()
+    # opener.addheaders = [('User-agent', 'Mozilla/5.0')]
+    # urllib.install_opener(opener)
+    
+    with PoolExecutor(8) as executor:
+        futures = []
+
+        N = (ex - sx + 1) * (ey - sy + 1)
```

Comment:
this seems to be unused

Reply:
Ah yes. Because I was using it to track progress in line 112
```py
print(f"{(100.0 * (i+1)/N):.2f}% ({request})")
```
but I ended up commenting it out. I'll comment this one out as well, but won't remove it since it might be useful at some point (and also if you're running it manually, you may uncomment them to check download progress)